### PR TITLE
Expose schema on ParquetFile

### PR DESCRIPTION
**Pull request by H-Plus-Time**

### CODE REVIEW

**Comment by H-Plus-Time** [comment]
This pairs well/best with the record_batches fix, do you want a bump of the arrow-wasm dep here (that or it's covered by one of the dep upgrade PRs)?

**Comment by kylebarron** [comment]
I bumped that here: https://github.com/kylebarron/parquet-wasm/pull/598/files#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542R64

File: src/reader_async.rs [hunk 1]
```diff
@@ -131,6 +131,11 @@ impl ParquetFile {
         Ok(self.meta.metadata().as_ref().to_owned().into())
     }
 
+    #[wasm_bindgen]
+    pub fn schema(&self) -> WasmResult<arrow_wasm::Schema> {
+        Ok(self.meta.schema().as_ref().to_owned().into())
```
Comment (kylebarron) [inline src/reader_async.rs:136]: Can you do `.clone()` instead of `as_ref().to_owned()`?

Comment (kylebarron) [reply src/reader_async.rs:136]: If `.schema()` gives you an `&Arc<Schema>`, then `.clone()` directly will just increase the reference count, while `.as_ref().to_owned()` will I think create a new `Schema`, copying all the underlying field data

**Review by kylebarron — APPROVED** [review]

**Pull request merged**